homeassistant/components/point/__init__.py

```diff
@@ -135,27 +184,35 @@ async def async_setup_webhook(hass: HomeAssistant, entry: ConfigEntry, session):
                 CONF_WEBHOOK_URL: webhook_url,
             },
         )
-    await session.update_webhook(
+
+    if await session.update_webhook(
```

Comment:
What can go wrong? And if it goes wrong, why do we delete the webhook altogether?

Reply:
I guess this can be moved to another pr.